src/fluree/db/ledger/txgroup/txgroup_proto.clj

```diff
@@ -45,42 +54,47 @@
   (async/<!! (kv-assoc-in-async group ks v)))
 
 (defn kv-dissoc-in-async
-  "Dissoc value as key sequence. Returns true if a value was present and removed, false if no value existed.
+  "Remove value as key sequence. Returns true if a value was present and removed, false if no value existed.
   Returns core async channel that will eventually have a response."
   [group ks]
   (let [command [:dissoc-in ks]]
     (-new-entry-async group command)))
 
 (defn kv-dissoc-in
-  "Dissoc value as key sequence. Returns true if a value was present and removed, false if no value existed."
+  "Remove value as key sequence. Returns true if a value was present and removed,
+  false if no value existed."
   [group ks]
   (async/<!! (kv-dissoc-in-async group ks)))
 
 (defn kv-max-in-async
-  "Writes value to specified key sequence. Will only write is proposed value is greater
-  than the current value. Returns true if value is written, false if it is not written.
-  Returns core async channel that will eventually have a response."
+  "Writes value to specified key sequence. Will only write is proposed value is
```

Comment:
Not yours, but I just spotted a typo here: "will only write is" -> "will only write if".